xbmc/input/InputManager.cpp

```diff
@@ -646,8 +646,9 @@ bool CInputManager::HandleKey(const CKey& key)
       action = m_buttonTranslator->GetAction(iWin, key);
   }
   if (!key.IsAnalogButton())
-    CLog::LogF(LOGDEBUG, "{} pressed, action is {}",
-               m_Keyboard.GetKeyName((int)key.GetButtonCode()), action.GetName());
+    CLog::LogF(LOGDEBUG, "{} pressed, window {}, action is {}",
+               m_Keyboard.GetKeyName((int)key.GetButtonCode()).c_str(), iWin,
```

Comment:
no need for .c_str()

Reply:
wow, 2 mistakes in a one-liner. A truly embarrassing case of copy-pasta-fail. Fixed.